dunwoody_disaster/views/MapScreen.py

```diff
@@ -127,6 +127,14 @@ def pixmap(self) -> QPixmap:
                 icon = QPixmap(DD.ASSETS["lock"]).scaledToWidth(100)
                 result = DD.overlay(result, icon, room)
 
+        for room in rooms["beaten"]:
+            icon = QPixmap(DD.ASSETS["completed"]).scaledToWidth(30)
+            result = DD.overlay(result, icon, room)
+
+        for room in rooms["all"] - rooms["beaten"]:
+            icon = QPixmap(DD.ASSETS["un-completed"]).scaledToWidth(30)
+            result = DD.overlay(result, icon, room)
```

Comment:
We probably shouldn't show the icon if there is an unbeaten boss and at least one unbeaten non-boss.